content/raw/chat/activity_center.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "activity center spec"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Activity Center
+
+## Designs
+
+- [designs](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=5358%3A5666)
+- [updated designs](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=1466%3A343105)
+
+## Use Cases
+
+TODO
+
+## Functional Requirements
+
+### General
+
+* The Activity Center MUST be displayed everywhere in the application
+* The Activity Center MUST support filtering by 5 categories: "All", "Mentions", "Replies", "Contact Requests", "Identity Checks"
+* The Activity Center MAY contain the following type of items:
+    * contact request
+    * mention in a community
+    * mention in public chat
+    * mention in a group
+    * mentions in a 1:1
+    * reply to a message from the user
+    * identity check request
+    * community invitation
```

Comment:
FYI, currently what is supported by status-go is:
- Mentions and replis **only** for group chats and communities
  - This was done to limit spam iirc. We wouldn't want spammers to spam your activity center with mentions
- **First** message from a **non** contact in a 1-1 chat
  - This is a way to support mutual contacts
  - Mentions and replies also don't appear from 1-1 chats, because 1-1 chats already send push notifications and act as mentions
- New group chat requests

That's it for now iirc. Someone from status-go can complement this comment.

One thing that was planned is to add actual contact requests in the activity center, but it requires a refactor, so it wasn't done yet.

Finally, I'm not sure what `identity check request`, could you explain it to me?